cwe_checker_rs/src/checkers/cwe_676.rs

```diff
@@ -0,0 +1,108 @@
+/// The CWE 676 checks for potentially dangerous function calls, such as strcpy, memcpy, scanf etc.
```

Comment:
The doc comment here should begin with `//!` (since it is inside the module which it documents) and not `///` (which needs to be outside of the object it documents. Also: We should insert the lengthy documentation for the CWE that we have in the corresponding `.mli` right now.